package.json

```diff
@@ -3,30 +3,34 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@craco/craco": "^6.2.0",
     "@reduxjs/toolkit": "^1.9.0",
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
-    "axios": "^1.1.3",
+    "axios": "^0.21.1",
+    "gh-pages": "^3.2.3",
     "prop-types": "^15.7.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-redux": "^7.2.9",
+    "react-redux": "^7.2.4",
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
     "redux": "^4.1.1",
     "redux-devtools-extension": "^2.13.9",
     "redux-logger": "^3.0.6",
-    "redux-promise-middleware": "^6.1.3",
     "redux-thunk": "^2.3.0",
+    "redux-toolkit": "^1.1.2",
     "uuid": "^8.3.2",
     "web-vitals": "^1.1.2"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "craco start",
+    "build": "craco build",
+    "test": "craco test",
```

Comment:
- [ ] I was unable to start up your project. See the error I get. Kindly fix that. I would recommend you stick to the CRA `react-scripts` commands.

  ![tyu](https://user-images.githubusercontent.com/54848363/201165309-b9882e45-6a25-407a-be38-6cee0d955bc8.jpg)

- [ ] This is what your app looks like after I change the commands to `react-scripts`. Kindly ensure your styling matches the design exactly after fixing the issues above.

  ![ryruow](https://user-images.githubusercontent.com/54848363/201166320-21ba8418-755f-4e77-93fd-7fc349247dd8.jpg)